manifests/a/AliceTeam/CarnegieMellonUniversity/Alice3/3.7.0.0.876/AliceTeam.CarnegieMellonUniversity.Alice3.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: AliceTeam.CarnegieMellonUniversity.Alice3
+PackageVersion: 3.7.0.0.876
+Installers:

```

```suggestion
AppsAndFeaturesEntries:
- DisplayName: Alice 3 3.7.0.0+build.876
  DisplayVersion: 3.7.0.0+build.876
Installers:
```
I suggested that the PackageVersion should be updated to use the "marketing" version. This version will be shown to the user for `winget install`. But WinGet requires the `DisplayVersion` written to Registry (that you can see in Control Panel/Apps&Features) for matching upgrades. As DisplayVersion != PackageVersion here (i.e., publisher markets the app with a different version that it writes to the registry), we can use `AppsAndFeaturesEntries` and explicitly provide DisplayVersion to help WinGet match upgrades.